server/common/points/get-points-leaderboard.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest } from "next";
+import { Sequelize } from "sequelize";
+
+import models from "db/models";
+
+import { lowerCaseCompare } from "helpers/string";
+
+export async function getPointsLeaderboard(req: NextApiRequest) {
+  const address = req.query?.address?.toString();
+
+  const users = await models.user.findAll({
```

Comment:
this will be very expensive;

Adding
```
  where: {
    [Op.or]: [
      { address },
      { address: { [Op.ne]: address } }
    ]
  },
  limit: 6
```

Will make sure we include the current user and limit the results to 6 users (in case the current user is not on the top five)

after doing `user = address ? users.find(u => lowerCaseCompare...`

we simply `users.pop()` the array to remove the 6th item and return 
```ts
{
  top: users // no need to slice this, as we limited to 6 and popped the last entry,
  user,
}
```